### ISSUE TITLE
Voila won't render if sidecar interactions

### ISSUE THREAD

**Issue by timkpaine** [QuantStack/voila#335]
xref: https://github.com/jupyter-widgets/jupyterlab-sidecar/issues/23

**Comment by SylvainCorlay** [comment]
Thanks. Good catch!

**Comment by jtpio** [comment]
Sidecar is quite specific to JupyterLab.

Do you think it should work transparently in Voila?

**Comment by timkpaine** [comment]
No, it should probably be a no-op. But right now it causes voila to not work at all, which means to take a jlab notebook with sidecar and make a Voila app requires code changes to the notebook.

**Comment by jtpio** [comment]
Yeah exactly, is should just display the output as normal.

**Comment by timkpaine** [comment]
@jtpio thats fine. Right now it causes voila to stop all rendering from that cell onward.
